app/assets/javascripts/discourse/app/routes/user-notifications.js

```diff
@@ -2,6 +2,9 @@ import DiscourseRoute from "discourse/routes/discourse";
 import ViewingActionType from "discourse/mixins/viewing-action-type";
 
 export default DiscourseRoute.extend(ViewingActionType, {
+  controllerName: "user-notifications",
```

Comment:
Is this needed? I think it should automatically use this controller because it has the same name 🤔

Reply:
If i remove this it gives this error
```
Assertion Failed: You're not allowed to have more than one controller property map to the same query param key, but both `userNotifications:filter` and `user-notifications:filter` map to `filter`. You can fix this by mapping one of the controller properties to a different query param key via the `as` config option, e.g. `filter: { as: 'other-filter' }
```